app/src/main/java/fr/free/nrw/commons/upload/MultipleShareActivity.java

```diff
@@ -233,6 +237,43 @@ protected void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    private void initPermissionsRationaleDialog() {
```

Comment:
Please write Javadocs for all new methods, thanks!

Reply:
Sure will add it

Reply:
Thanks for the detailed Javadocs! 👍